File: music-api/src/main/java/com/music/quiz/model/QuizResponse.java
package com.music.quiz.model;

public record QuizResponse(
        Long userId,
        Long firstTrackId,
        String firstTrackTitle,
        Long secondTrackId,
        String secondTrackTitle,
        Long thirdTrackId,
        String thirdTrackTitle,
        String words,
        Long correctAnswerId,
        Long userAnswerId
) {
}

File: music-angular/src/app/services/quiz/models/quiz-response/quiz-response.ts
export class QuizResponse {
  constructor(
    public userId: number,
    public firstTrackId: number,
    public firstTrackTitle: string,
    public secondTrackId: number,
    public secondTrackTitle: string,
    public thirdTrackId: number,
    public thirdTrackTitle: string,
    public words: string,
    public correctAnswerId: number,
    public userAnswerId: number
  ) {
  }
}
